data/keys

```diff
@@ -7,10 +7,10 @@ Global {
 # Adding your own frequently used actions is easy -
 # just copy it over from CHAINS and edit the keypress!
 	# Moving in frames
-	KeyPress = "Mod1 Tab" { Actions = "NextFrame EndRaise" }
-	KeyPress = "Mod1 Shift Tab" { Actions = "PrevFrame EndRaise" }
-	KeyPress = "Mod1 Ctrl Tab" { Actions = "NextFrameMRU EndRaise" }
-	KeyPress = "Mod1 Ctrl Shift Tab" { Actions = "PrevFrameMRU EndRaise" }
+	KeyPress = "Mod4 Tab" { Actions = "NextFrameMRU TempRaise" }
```

Comment:
I prefer MRU in general, that's why I changed the actions of the 4 rows.
I also find TempRaise very convenient because it raises the window so you see which window would be focused